docs/docs/integrations/providers/qdrant.mdx

```diff
@@ -13,6 +13,27 @@ Install the Python partner package:
 pip install langchain-qdrant
 ```
 
+## Embedding models
+
+### FastEmbedEmbeddings
+
+See a [usage example](/docs/integrations/text_embedding/fastembed).
+
+```python
+from langchain_community.embeddings.fastembed import FastEmbedEmbeddings
```

Comment:
remove and happy to deprecate in favor of langchain_qdrant ones

Reply:
Yeah, I do not add description of classes that marked with the "deprecated" decorator. This class was not attributed with the decorator, so I was confused :)